build.gradle.kts

```diff
@@ -1,11 +1,10 @@
+@file:Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
 plugins {
-  //trick: for the same plugin versions in all sub-modules
-  id("com.android.application").version("7.4.0").apply(false)
-  id("com.android.library").version("7.4.0").apply(false)
-  kotlin("android").version("1.8.0").apply(false)
-  kotlin("multiplatform").version("1.8.0").apply(false)
-}
-
-tasks.register("clean", Delete::class) {
-  delete(rootProject.buildDir)
+    alias(libs.plugins.android.application) apply false
+    alias(libs.plugins.android.library) apply false
+    alias(libs.plugins.kotlin.android) apply false
+    alias(libs.plugins.kotlin.multiplatform) apply false
+    alias(libs.plugins.kotlin.serialization) apply false
+    //alias(libs.plugins.kotlin.kapt) apply false
+    //alias(libs.plugins.ksp) apply false
```

Comment:
@oneSIX I commented these out for now as they aren't currently used, do we want to keep them in/intend to use them?

Reply:
I think this was just some leftover copy/pasta from when I was setting it up. We will probably end up using KSP later but for now i'll remove this.